### ISSUE TITLE
Contact Form.php Not working

### ISSUE THREAD

**Issue by abhibhaw** [abhibhaw/abhibhaw.github.io#5]
# I have used the mail() function of PHP to send emails on form submission but it's not working.

Used Bootstrap V5.0 for frontend and looking for some js or PHP for emailing the contact submission.

**Comment by abhibhaw** [comment]
flowchart TD;

start((Create a branch))

Create a branch to work on your changes.

Branch((Commit your changes))

Commit your changes to your branch.

Push((Create a pull request))

Push your branch to the remote repository and create a pull request.

Review((Review and approve))

Other developers will review your pull request and approve it if it is ready to be merged.

Merge((Merge))

Once your pull request is approved, it will be merged into the main branch.

end((Done))
